deployment/frontend/next.config.mjs

```diff
@@ -12,6 +12,8 @@ const config = {
     ignoreDuringBuilds: true,
   },
   reactStrictMode: true,
+  output: "standalone",
```

Comment:
does `output: "standalone",` works with `npm run start` use to think this creates a server.js that should be run with `npm server.js` or so

Reply:
true, my mistake, forgot to checkout that file